src/containers/Careers.js

```diff
@@ -15,25 +19,122 @@ const CareersContainer = styled.div`
   flex-direction: column;
 `;
 
+const SwingWrapper = styled.div`
+  position: relative;
+  height: 70%;
+  width: 80%;
+  margin: 0 2rem;
+  z-index: 0;
+  overflow: visible;
+`;
+
+let swipeCount = 1;
+
 class Careers extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      stack: null,
+    };
+  }
+
   componentDidMount() {
     this.props.fetchCareers();
   }
 
+  swingShow = () => {
+    if (
+      this.props.careers.dataFetched &&
+      this.props.careers.careerCards.length
+    ) {
+      return (
+        <Swing
+          tagName="div"
+          setStack={stack => this.setState({ stack: stack })}
+          ref="stack"
+          throwout={e => {
+            swipeCount++; // someone please suggest a better way to do this
+            this.likeCareer(e);
+          }}
+          config={{
+            allowedDirections: [Direction.LEFT, Direction.RIGHT],
+            maxRotation: 5,
+            minThrowOutDistance: 1000,
+            maxThrowOutDistance: 1400,
+            throwOutConfidence: (xOffset, yOffset, element) => {
+              const xConfidence = Math.min(
+                Math.abs(xOffset) / element.offsetWidth,
+                1
+              );
+              const yConfidence = Math.min(
+                Math.abs(yOffset) / element.offsetHeight,
+                1
+              );
+              if (Math.max(xConfidence, yConfidence) > 0.65) {
+                return 1;
+              }
+              return Math.max(xConfidence, yConfidence);
+            },
+          }}
+        >
+          {this.props.careers.careerCards.map(card => (
+            <div
+              className="cardwrapper"
+              ref={`card${card.id}`}
+              id={`card${card.id}`}
+              key={card.id}
+            >
+              <CareerCard card={card} />
+            </div>
+          ))}
+        </Swing>
+      );
+    }
+    // TODO render loading
+    return;
+  };
+
+  likeCareer = e => {
+    if (e.throwDirection.toString() === 'Symbol(RIGHT)') {
+      return this.props.likeCareerAction({
+        user_id: 1,
+        career_id: Number(e.target.id.slice(4)),
+      });
+    }
+  };
+
+  throwCard = direction => {
+    const cardRef = Object.keys(this.refs.stack.refs)[
```

Comment:
As discussed in person, clear up the variable naming so it's a bit clearer what's happening here and you don't make duplicate calls to Object.keys, etc.